tsconfig.json

```diff
@@ -0,0 +1,33 @@
+{
+  "compilerOptions": {
+    /* Basic Options */
+    "target": "ES5",
+    "module": "CommonJS",
+    "lib": ["ES2018"],
+    // "allowJs": true,
+    // "checkJs": true,
+    "declaration": true,
+    "outDir": "./build/",
+    "noEmit": true,
+    "isolatedModules": true,
+
+    /* Strict Type-Checking Options */
+    "strict": true,
+
+    /* Additional Checks */
+    "noUnusedLocals": true,
+    "noImplicitReturns": true,
+    "noFallthroughCasesInSwitch": true,
+
+    /* Module Resolution Options */
+    "moduleResolution": "node",
+    // "allowSyntheticDefaultImports": true,
+    "esModuleInterop": true,
+
+    /* Advanced Options */
+    "skipLibCheck": true,
```

Comment:
is this needed?

Reply:
I'm not sure. It's enabled in tsconfig.json when I ran `tsc --init`. I think we can remove it.